### PR TITLE
migrate NuGetGallery.Core.Facts to sdk csproj 

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: tests/NuGetGallery.Core.Facts/NuGetGallery.Core.Facts.csproj [hunk 1]
```diff
@@ -1,177 +1,27 @@
...
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <Prefer32Bit>false</Prefer32Bit>
+  <PropertyGroup>
```
Comment (joelverhagen) [inline tests/NuGetGallery.Core.Facts/NuGetGallery.Core.Facts.csproj:5]: Same comment as https://github.com/NuGet/NuGetGallery/pull/9990/files#r1627810144

File: tests/NuGetGallery.Core.Facts/NuGetGallery.Core.Facts.csproj [hunk 2]
```diff
@@ -1,177 +1,27 @@
...
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <Prefer32Bit>false</Prefer32Bit>
+    <TargetFramework>net472</TargetFramework>
   </PropertyGroup>
+  <Import Project="$(SignPath)\sign.targets" Condition="Exists('$(SignPath)\sign.targets')" />
   <ItemGroup>
-    <Reference Include="System" />
     <Reference Include="System.ComponentModel.DataAnnotations" />
```
Comment (erdembayar) [inline tests/NuGetGallery.Core.Facts/NuGetGallery.Core.Facts.csproj:7]: Could we remove remainin System references?

File: tests/NuGetGallery.Core.Facts/NuGetGallery.Core.Facts.csproj [hunk 3]
```diff
@@ -1,177 +1,24 @@
...
-      <ExcludeAssets>analyzers</ExcludeAssets>
+    <ProjectReference Include="..\..\src\NuGet.Services.Entities\NuGet.Services.Entities.csproj" />
+    <ProjectReference Include="..\..\src\NuGetGallery.Core\NuGetGallery.Core.csproj" />
+
+    <PackageReference Include="Moq" Version="4.8.2" />
+    <PackageReference Include="xunit.abstractions" Version="2.0.3" />
+    <PackageReference Include="xunit.assert" Version="2.5.0" />
+    <PackageReference Include="xunit.core" Version="2.3.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.5.0">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
```
Comment (erdembayar) [inline tests/NuGetGallery.Core.Facts/NuGetGallery.Core.Facts.csproj:15]: `<ExcludeAssets>analyzers</ExcludeAssets>` needs to be separate from `IncludeAssets`

**Comment by erdembayar** [comment]
@SimonCropp 
Sorry, this one got merge conflict now.

**Comment by erdembayar** [comment]
Could you please address my comment above? If you are unable to work on this anytime soon, please close the and reopen it later when it's ready.

**Pull request closed without merging**